wagtail/contrib/styleguide/templates/wagtailstyleguide/base.html

```diff
@@ -7,578 +7,657 @@
     {{ example_form.media.css }}
 {% endblock %}
 
-{% block bodyclass %}styleguide{% endblock %}
+{% block bodyclass %}styleguide
+{% endblock %}
```

Comment:
This content appears to have been accidentally formatted to two lines. 

Please check your editor has got the correct settings for Prettier maybe.

Reply:
I'm going to fix the formatting issues on my editor.

Reply:
What editor are you using?

Reply:
VS code, surprisingly.

Reply:
Here's the docs for some settings that may help. 

https://code.visualstudio.com/docs/languages/html#_formatting

Reply:
Thanks!! Checking it right now.